ts/features/wallet/onboarding/bancomat/navigation/action.ts

```diff
@@ -3,6 +3,11 @@ import { InferNavigationParams } from "../../../../../types/react";
 import { ActivateBpdOnNewCreditCardScreen } from "../../common/screens/bpd/ActivateBpdOnNewCreditCardScreen";
 import WALLET_ONBOARDING_BANCOMAT_ROUTES from "./routes";
 
+export const navigateToOnboardingBancomatChooseBankInfo = () =>
+  NavigationActions.navigate({
+    routeName: WALLET_ONBOARDING_BANCOMAT_ROUTES.CHOOSE_BANK_INFO
+  });
+
```

Comment:
Like the route, what you think about renaming this action to `navigateToOnboardingBancomatSearchStartScreen` or something similar?